src/runtime_src/core/tools/xbutil2/SubCmdReset.cpp

```diff
@@ -20,6 +20,7 @@
 #include "SubCmdReset.h"
 #include "core/common/query_requests.h"
 #include "tools/common/XBUtilities.h"
+#include "core/pcie/driver/linux/include/mailbox_proto.h"
```

Comment:
this is common code for pcie and edge devices. We can't have pcie specific include files.

Reply:
got it.
move the changes into xocl driver